src/requestLogger.js

```diff
@@ -7,9 +7,11 @@ class RequestLogger {
    * @param {Function} configuration.logFunction optional log function, by default RequestLogger uses console.log
    * @param {Boolean}  configuration.extendErrorObjects extends Error object globally in order to provide proper JSON
    *                   representation of Error objects.
+   * @param {Number}   configuration.jsonSpace the number of spaces that are used then stringifying the message.
    */
-  constructor(configuration = { logFunction: console.log, extendErrorObjects: true }) {
+  constructor(configuration = { logFunction: console.log, extendErrorObjects: true, jsonSpace: 2 }) {
     this.logFunction = configuration.logFunction;
+    this.jsonSpace = configuration.jsonSpace;
```

Comment:
we actually need the default here `configuration.jsonSpace` or this will be a regression for every consumer of the `constructor` which is passing in an object here.

Reply:
Good point. Added.